run-elrond-tests.py

```diff
@@ -162,20 +172,22 @@ def convert_string_to_uint(raw_str: str):
 
     if num_str.startswith('0x') or num_str.startswith('0X'):
         num_str = num_str[2:]
-        num_len = len(num_str) // 2
-        if num_len == 0:
+        str_len = len(num_str)
+        if str_len == 0:
```

Comment:
Does this mean that we can encounter hex strings with only one char? Like `0xf`?

Reply:
we'll left pad 0. that's why `num_len = (str_len + 1) // 2`